data/json/npcs/cabin chemist/chemist npc.json

```
@@ -0,0 +1,259 @@
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_CABIN_CHEMIST_INTRO",
+    "dynamic_line": "So, what's up?",
+    "responses": [
+      { "text": "Who are you?", "topic": "TALK_NPC_CABIN_CHEMIST_STORY" },
+      { "text": "What is this place?", "topic": "TALK_NPC_CABIN_CHEMIST_HOME" },
+      { "text": "Care to trade?", "topic": "TALK_NPC_CABIN_CHEMIST_INTRO", "effect": "start_trade" },
+      {
+        "text": "I'd like to buy in bulk.",
+        "topic": "TALK_CABIN_CHEMIST_BULK_SELL",
+        "condition": { "npc_has_var": "mass_chem_tradeable", "type": "dialogue", "context": "trade", "value": "yes" }
+      },
+      {
+        "text": "Any jobs you need done?",
+        "condition": { "and": [ { "not": "has_assigned_mission" }, { "not": "has_many_assigned_missions" } ] },
+        "topic": "TALK_MISSION_LIST"
+      },
+      { "text": "I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_CABIN_CHEMIST_STORY",
+    "dynamic_line": "I used to be a chemist.  Well, I am a chemist.  I've been living out here in the wilderness for a little while now since <the_cataclysm>.  Even brought some of my equipment, though unfortunately missing some of the more sensitive and expensive stuff.  I found a couple useful things here in the cupboards, but other than that, entirely empty.  So, I moved in.",
+    "responses": [ { "text": "Hmm.", "topic": "TALK_NONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_CABIN_CHEMIST_HOME",
+    "dynamic_line": "Not my place, just somewhere I ended up.  Some old shack, maybe a hunting cabin or something more… criminal, at least judging by some of what I found on the shelves.  You certainly wouldn't attract much attention synthesizing illegal substances all the way out here, that's all I'm saying.  I found it while I was trying to escape all the <zombies> in the city, and I decided to stay.",
+    "responses": [ { "text": "Hmm.", "topic": "TALK_NONE" } ]
+  },
+  {
+    "id": "TALK_CABIN_CHEMIST_BULK_SELL",
+    "//": "This is all the dialogue related to buying bulk chemicals.  It should remain somewhat prohibitively expensive and time-consuming so the chemist doesn't become a one-stop shop for survival needs. TO-DO - Use variables to refine shop system. See SCRIPT_EFFECTS.md and isolated road fabriation system.",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "npc_has_effect": "currently_busy",
+      "yes": "I'm out of some things, <name_g>.  It takes a while to source the good stuff - days usually - but I'm working hard to replace it.  What can I get you in the meantime?",
+      "no": "I'll see what I've got around here for you.  I've gotta warn you: it's gonna be expensive.  Hard to stay supplied in the apocalypse, you know.  What do you need?"
+    },
+    "responses": [
+      { "text": "Nevermind.", "topic": "TALK_NONE" },
+      {
+        "text": "I'll buy some bandages.  [10$]",
```

These currencies should all be moved to the beginning of the line for consistency
```suggestion
        "text": "[$10] I'll buy some bandages.",
```

---

Please make sure you apply this to all of the others as well.